client/src/components/History/History.module.scss

```diff
@@ -16,10 +16,11 @@
   flex-grow: 1;
 }
 
+/* WP-50: Font-size below was overwriting previous font-size establishments. This should now be rectified. */
 .header {
   button {
     padding: 0;
-    font-size: 0.78em;
+    /* font-size: 0.78em; */
```

Comment:
```suggestion

```

Reply:
Instead of commenting out the code, feel free to delete it completely.

Reply:
And now I ask myself, "why _did_ I leave that in?"
Done and done!